Formula/o/openvino.rb

```diff
@@ -21,7 +21,10 @@ class Openvino < Formula
     sha256 cellar: :any,                 arm64_ventura: "dbb9da7ce2c386c0d28b3349d01524b3298afdf46a0c1be8f7c7ef9bf0c65e60"
     sha256 cellar: :any,                 sonoma:        "6443b56a2b56b273f04a716e87dc25d96e486f3955733631c2ac5c92dab80f94"
     sha256 cellar: :any,                 ventura:       "ec5ed79b6a10cf6dab861e3eafe3fdc925ce9fdaafb8a89e7cca6e379fe8e10a"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:  "db2178c8c3450d514c0ec118be1e84710cde5c516ea0c3c32c6ae8b458287394"
+    # Temporarily disable Linux platform due to build errors.
+    # It's preferable to unblock macOS users with a new release
+    # rather than delay it for everyone. Re-enable once the issue
+    # is resolved in Homebrew (likely with a protobuf module bump).
```

Comment:
If there is no openvino for linux, opencv won't exist for linux either. And with 6k installs in the past month that seems risky. Is there really no way to fix the linux build now?

Either way, we should add `depends_on :macos` if we want to drop Linux support instead of removing the bottle, because the bottle will be re-added on merge.

Reply:
> Is there really no way to fix the linux build now?
I assume that it's not connected to our changes, but to protobuf(check my comment above)
The same errors in opencv PR btw - https://github.com/Homebrew/homebrew-core/pull/228841

 

>Either way, we should add depends_on :macos if we want to drop Linux support instead of removing the bottle, because the bottle will be re-added on merge.

Yeah, make sense.
should I add it to opencv recipe only?

Reply:
I just realised opencv is quite a popular dependency so this would break a bunch of dependents on Linux. I think we need to wait for a Linux fix.

Reply:
Yeah, because for now even if I'll update opencv formulae with `depends_on :macos` for openvino module - the CI still tries to build opencv on linux because of revision bump... and then we got again error connected to protobuf.